modulefiles/common-run.lua

```diff
@@ -0,0 +1,10 @@
+help([[
+Load common modules to run GSI Monitoring on all machines
+]])
+
+local wgrib2=os.getenv("wgrib2") or "2.0.8"
+local prod_util=os.getenv("prod_util") or "1.2.2"
```

Comment:
These should be wgrib2_ver and prod_util_ver.

Reply:
Yup, fixed.